apps/base-docs/docs/building-with-base/guides/hyperframes.md

```diff
@@ -2,10 +2,17 @@
 title: 'HyperFrames'
 description: Learn how to make cross-linked hyperframes in an organized manner.
 hide_table_of_contents: false
+keywords: [farcaster, frames, farcaster frames, hyperframes, hyper frames, state, frame state, Base]
 ---
 
 [Frames] on [Farcaster] are getting more complex. Developers are now building interactions that require a handful, or even dozens of frames in response to various user states, inputs, and actions. HyperFrames are a system to organize navigation for large numbers of frames, using [OnchainKit]. In this tutorial, we'll use making the navigation for an [old-school adventure game] fully in a frame. You can use this same technique for many other intents with your Frames, such as games, stores, customized mints, etc.
 
+:::info
```

Comment:
Idk if this call out is necessary. How far in the future would we remove it?

Reply:
I'll delete it.  It's redundant with the warning that these are in flux now anyway.